app/models/messaging/bsnl/dlt_template.rb

```diff
@@ -108,4 +131,12 @@ def resize_variables_equally(permitted_length, content)
       [k, v[0, lengths[k]]]
     end
   end
+
+  class << self
+    def numeric?(string)
```

Comment:
Wanna just declare `def self.numeric?`, consistent with the other class methods in here?

Reply:
standardrb enforces this:
```
Lint/IneffectiveAccessModifier: `private` (on line 102) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.
```

Reply:
But we don't need to make this private, yeah? We could just do it the other way and move the method above the `private`?

Reply:
Wanted to make it private since no one should need to call `DltTemplate.numeric?` ever.